content/news/playtest-20151122.md

```diff
@@ -0,0 +1,17 @@
+We received a lot of feedback on our last two playtests, and we tried to address as many of the reported issues as possible.  Thanks to everyone who has tried the playtests and reported bugs or other feedback!
+
+We fixed some new, as well as some long-standing issues for this playtest, so hopefully this will be the final release candidate for the upcoming December release.  We encourage you to give this release a good try out, especially if you like to play competitively online, because if we don't hear your balance complaints now then you will be stuck with them for the next few months until our next release!  You can report any issues you have by [filing a bug](http://bugs.openra.net) or [joining our IRC channel](/community/).
+
+The focus for this playtest was around:
+* Improving targeting logic.
```

Comment:
You need a newline before you start a list, otherwise the markdown parser doesn't recognize it.